cmd/ci-operator/main.go

```diff
@@ -483,6 +483,7 @@ func bindOptions(flag *flag.FlagSet) *options {
 }
 
 func (o *options) Complete() error {
+	o.resolveConsoleHost()
```

Comment:
I don't think this is sufficient to fully fix DPTP-2760. There are still cases (when `resolveConsoleHost` fails mostly) when `consoleHost` is empty, and then the report will still fail.

This patch is good, but I think we should also augment the actual reporter to handle the case of `consoleHost` being empty by e.g. putting an `unknown` there:

https://github.com/openshift/ci-tools/blob/e1f2f0d34b7c2697040c954abc696302671bb729/pkg/results/report.go#L46-L64